.jenkins/python.yaml

```diff
@@ -2,17 +2,41 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: node
+  annotations:
+    container.apparmor.security.beta.kubernetes.io/img: "unconfined"
+    container.seccomp.security.alpha.kubernetes.io/img: "unconfined"
 spec:
+  volumes:
+    - name: docker-socket-volume
+      hostPath:
+        path: /var/run/docker.sock
+        type: File
   containers:
     - name: python
       image: python:3-alpine
       tty: true
+      command:
+        - cat
+      resources:
+        requests:
+          memory: '512Mi'
+          cpu: '1'
+        limits:
+          memory: '512Mi'
+          cpu: '1'
+      volumeMounts:
+        - mountPath: /var/run/docker.sock
+          name: docker-socket-volume
+    - name: img
+      image: r.j3ss.co/img
```

Comment:
what are the pros of using img vs the jenkins library?

Reply:
copy-paste from k8t
I'd rather use dockerhub autobuild but I have no admin access to it